.github/workflows/build-cache.yml

```diff
@@ -39,6 +41,22 @@ jobs:
 
     - name: Save date of cache build
       run: date -R -u > target/cache-build-date.txt
+
+    # It seems that in an attempt to be more compatible with GitHub, WarpBuild removed
+    # the ability to overwrite an old cache and they longer charge for cache storage.
```

Comment:
"no longer"?

Reply:
No. I misread the billing statement. They are still charging it.

They just removed the ability to automatically replace a cache if I try to save with an identical key.

Reply:
removed comment